analysis/diabetes_analysis.ipynb

```diff
@@ -2089,13 +2140,39 @@
     "\n",
     "Moreover, there runs the possibility that the features offered from this dataset alone are not sufficient to predict with high accuracy. In this case, conducting additional conversations with data collectors for additional useable information or explore additional datasets that can be joined so our set of features can be expanded for more complicated analysis might be beneficial. "
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "973fcb4d-ee7e-4367-a0d6-8e674bf4eb85",
+   "metadata": {},
+   "source": [
+    "# References"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "13a22e58-2428-4858-bef1-c90c600813d1",
+   "metadata": {},
+   "source": [
+    "Agarwal, N., & Vadiwala, R. (2022). Machine Learning and Data Mining Methods in Diabetes Research. Asian Journal of Organic & Medicinal Chemistry.\n",
```

Comment:
No issues with references. 

Just may need to update / delete / add per changes made in introduction.

Reply:
minor update to match changes in the narrative